### DISCUSSION TITLE
Passing extra parameter except df to the function running on tabpy server.

### DISCUSSION THREAD

**Discussion by ashu1click** [tableau/TabPy#561] [Q&A] [answered]
I am working on one scenario in which I have one python script that is running on TabPy Server
Ex - getData(df) , here df is coming from one excel in the Tableau Prep Builder , that is working as connection. 
In the python script I am hitting one API, I want to change the one parameter of the API dynamically. I am not able to find out any way to traverse a parameter to the python script coming through Tableau Prep Builder.
Any clue is it possible or not?

**Comment by nmannheimer** [comment, marked as answer]
Unfortunately Prep doesn't support dynamic parameters that can be changed automatically. 

You could code this into your script though and call some list of values.
